### PR TITLE
[TS] LPS-180053 Add a null check to _imageAlt to prevent the NPE in PortlettRequestDispatcherImpl which causes web content template portlet failure

### PR DESCRIPTION
**Pull request by evibudai**
Hi Team,

https://issues.liferay.com/browse/LPS-180053

This issue is only reproducible on a JBOSS/Wildfly bundle.

Issue:
The web content template portlet is unavailable, user cannot see the templates. It happens because of a nullPointerException in PortlettRequestDispatcherImpl, which is caused by a missing _imageAlt. (if you set the log level to DEBUG for PortletRequestDispatcherImpl class, you'll see a NPE)

https://user-images.githubusercontent.com/97447507/231758745-d09b19da-376a-4231-880c-f599afd88968.mp4

Solution:
Based on Fabian Bouché's suggestion I added a null check to _imageAlt, which solved the issue:

https://user-images.githubusercontent.com/97447507/231757891-087c5a9c-e433-41a8-8914-207703a4d6d8.mp4

I tested it on master in my local environment on JBOSS bundle, and the result can be seen in the above video.

Best regards,
Évi

### CODE REVIEW

**Comment by evibudai** [comment]
ci:test:sf

**Comment by dsanz** [comment]
Hi @evibudai, I have some doubts about this fix.

First, we are emiting a non-null `alt` for the image in the Sticker, even those cases where no `alt` attribute has been indicated. This changes the semantics of the image to the eyes of the screen reader, which now will announce the image alternative text as "UNDEFINED". This surely will come back to us in the form of an accessibility bug.

Second, I don't see how setting a non-null `alt` attribute prevents jboss body writer to work. This looks like a sign of a more underlying cause that should be described in the ticket. That is the only way we can figure it out the proper solution.

According to Fabian, 

> The root cause was that some users who created the templates had a portraitId with the ID of a missing document in the document library.

However, in the [LPP](https://issues.liferay.com/browse/LPP-48382?focusedCommentId=2875247&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2875247), Tibor said that the document is not missing, which sounds like the original root cause was not really causing this.

So, could you please help us to understand what is causing this?

For the purposes of this pull, I'm going to assume that `jspWriter.write(_imageAlt);` in StickerTag is bothering JBoss when a `null` value is passed. This is not an explanation but a working hypothesis.

Ideally, this shall be controlled by the tag making use of user information, so that they can properly invoke StickerTag. However, it seems we're [already doing it](https://github.com/liferay/liferay-portal/blob/e1fb9f0e12469c59e684e9ab6a7ff866c6d89435/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java#L503-L505). Still, original value for the `_imageAlt` can be null if not set, so it's fine to check for null in the StickerTag as Fabian suggests.

So, in short, we'd need to know two things:
- Though not strictly necessary, how this connects with the JBoss error. We'd like to make sure what the cause is in case we have to review other tags.
- The reason why do we need to emit "UNDEFINED" when there is no image alt. If tag caller can't figure it out what the alt shall be, then we can leave that empty under the assumption of it being a [purely decorative image](https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information). Probably, removing the `else` clause would be enough.

Thanks a lot

**Comment by fabian-bouche-liferay** [comment]
When I diagnosed the issue on the customer's infra, they told me they did not import the document library in their test environment. That's why I assumed that was the root cause.
However, Tibor observed that the absence of the image wasn't even necessary.

Instead of putting UNDEFINED, we could just output some language key which means "portrait" as the picture alt since the picture contains the portrait of the user.

We could issue this as a short term fix for the customer who's blocked by this bug and indeed carry on with more investigation because there might be a more profund root cause in JBoss which could cause issues with other tags.

**Comment by evibudai** [comment]
Hi @dsanz , @fabian-bouche-liferay 

I verified that with the `else` clause removed, the fix is still working.

**Comment by dsanz** [comment]
Hi @fabian-bouche-liferay @evibudai ,

removing the `else` clause looks a reasonable solution to me from the pov of the product, I mean, this can be left there and we can move on with the investigation in case other stuff comes up with this customer.

Thanks a lot

**Comment by dsanz** [comment]
ci:forward
